cognite/client/testing.py

```diff
@@ -53,8 +59,10 @@ def __init__(self, *args, **kwargs):
         self.raw.databases = MagicMock(spec_set=RawDatabasesAPI)
         self.raw.tables = MagicMock(spec_set=RawTablesAPI)
         self.raw.rows = MagicMock(spec_set=RawRowsAPI)
+        self.relationships = MagicMock(spec_set=RelationshipsAPI)
         self.sequences = MagicMock(spec=SequencesAPI)
         self.sequences.data = MagicMock(spec_set=SequencesDataAPI)
+        self.synthetic_time_series = MagicMock(spec_set=SyntheticDatapointsAPI)
```

Comment:
this is not a thing
https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_cognite_client.py

Reply:
Ah I see. I guess the same goes for `EntityMatching`